llvm/test/Transforms/LoopVectorize/AArch64/prefer-fixed-if-equal-to-scalable.ll

```diff
@@ -0,0 +1,196 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
I wonder if it's necessary to run `utils/update_test_checks.py` on this test, since all you're really testing is that we vectorised with a particular VF. For example, you could reduce the number of CHECK lines in each function to just the bare minimum:

```
; CHECK-LABEL: define void @NeoverseV2(
: CHECK:    store <4 x float>
```